packages/web/src/pages/collectibles-playlist-page/CollectiblesPlaylistPageProvider.tsx

```diff
@@ -188,14 +189,14 @@ export const CollectiblesPlaylistPageProvider = ({
           } else {
             const a = new Audio()
             const duration: Promise<number> = new Promise((resolve) => {
-              setTimeout(() => resolve(0), 60000)
+              setTimeout(() => resolve(0), 4000)
```

Comment:
why this

Reply:
dont need the change, but waiting 60 s just to get duration seems excessive